pydm/data_plugins/local_plugin.py

```diff
@@ -89,43 +98,51 @@ def convert_value(self, value, value_type):
         if value_type == 'int':
             try:
                 return int(value)
-            except TypeError:
+            except ValueError:
                 pass
         elif 'ndarray' in value_type:
             try:
                 # evaluate it first
                 value_list = ast.literal_eval(value)
+                value_array = None
                 # convert into a numpy array
-                value_array = np.array(str(value_list))
+                if subtype is not None:
```

Comment:
As soon as I added the `subtype` I broke the display of `ndarray` in an `EditLine` again, so I had to send it as a list to make it work..... but I am not too happy about it. 
The problem is that when we go through the `EditLine` widget, we convert things to string, an then it is very difficult to convert it back to a proper `ndarray` if we don't know too much about it, please let me know if you have any suggestions about this.